How to get started for contributing?

I'd like to help on improving the training docs, but there are no instructions in this repo for getting started, including (but not limited to):

1. [x] [How to contribute](https://github.com/blog/1184-contributing-guidelines) See #238
1. [x] Installation requirements
1. [x] How to build the docs and verify locally before making a PR

Perhaps some stuff from [Pyramid](https://github.com/Pylons/pyramid) could be useful?

For 2 and 3, I did the following, but got stuck when the error appeared. I think I need an assist to install pyenchant (or enchant) and whatever else it needs.

```bash
# after creating a Python 2.7.12 virtual environment
$ env27/bin/pip install --upgrade pip wheel setuptools
$ env27/bin/pip install -q -r requirements.txt --use-mirrors
# no such option: --use-mirrors
$ env27/bin/pip install -q -r requirements.txt
$ make html SPHINXBUILD=env27/bin/sphinx-build

/Users/stevepiercy/projects/training/env27/bin/sphinx-build -b html -d _build/doctrees   . _build/html
Running Sphinx v1.4.6
making output directory...

Exception occurred:
  File "/Users/stevepiercy/.pyenv/versions/2.7.12/lib/python2.7/ctypes/__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
OSError: dlopen(/Users/stevepiercy/projects/training/env27/lib/python2.7/site-packages/enchant/lib/libenchant.1.dylib, 6): Symbol not found: ___strlcat_chk
  Referenced from: /Users/stevepiercy/projects/training/env27/lib/python2.7/site-packages/enchant/lib/libglib-2.0.0.dylib
  Expected in: /usr/lib/libSystem.B.dylib
 in /Users/stevepiercy/projects/training/env27/lib/python2.7/site-packages/enchant/lib/libglib-2.0.0.dylib
The full traceback has been saved in /var/folders/8p/jlq9d1y13ss8q9hb744r6_fr0000gn/T/sphinx-err-eYfOd6.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
make: *** [html] Error 1
```

If I comment out anything to do with spelling in `conf.py`, then I can build the docs.

--- a/CHANGES.txt
+++ b/CHANGES.txt
@@ -5,6 +5,9 @@ This changelog is only very rough. For the full changelog please refer to https:
 
 1.2.5 (unreleased)
 ------------------
+- Update README.rst to refer to how to build the docs locally.
+  [stevepiercy]
+
 - Add CONTRIBUTING.md
   [stevepiercy]
 

--- a/README.rst
+++ b/README.rst
@@ -37,4 +37,5 @@ If you are having issues, please let us know by opening an issue in our `Issue T
 License
 -------
 
-The project is licensed under the GPLv2.
+The project is licensed under the `Creative Commons Attribution 4.0 International License <http://creativecommons.org/licenses/by/4.0/>`_ by the `Plone Foundation <https://plone.org>`_
+

--- a/README.rst
+++ b/README.rst
@@ -1,7 +1,40 @@
-Mastering Plone
-===============
+Training
+========
 
-Read the trainings at http://training.plone.org
+`Training <https://gihub.com/plone/training>`_ is a collection of different trainings, developed and created by the Plone Community.
 
-Build the training documentation locally:
-https://training.plone.org/5/mastering_plone/about_mastering.html#building-the-documentation-locally
+For a HTML version, please browse to our `Training Website <https://training.plone.org>`_.
+
+Features
+--------
+Different Plone Trainings:
+
+- `How to work with content and manage site settings in Plone. <https://training.plone.org/5/>`_
+- `How to get you Plone site up and running. <https://training.plone.org/5/deployment/index.html>`_
+- `How to develop customized solutions with Plone. <https://training.plone.org/5/mastering_plone/index.html#mastering-plone-label>`_
+- `How to add enterprise grade search to your Plone site. <https://training.plone.org/5/solr-training/index.html>`_
+- `How to style your Plone site. <https://training.plone.org/5/theming/index.html>`_
+- `Create custom content types, a design for a website, layouts for homepages and content types, and custom application logic. All in the browser. <https://training.plone.org/5/ttw/index.html>`_
+
+Documentation
+-------------
+
+Documentation on how to use, build and contribute to the trainings can be found on the `Training Website <https://training.plone.org/5/about.html>`_ .
+
+Contribute
+----------
+
+- `Issue Tracker <https://github.com/plone/training/issues>`_
+- `Source Code <https://github.com/plone/training>`_
+- `Website <https://training.plone.org/>`_
+
+Support
+-------
+
+If you are having issues, please let us know by opening an issue in our `Issue Tracker <https://github.com/plone/training/issues>`_ or asking a question on our
+`Community Space <https://community.plone.org>`_.
+
+License
+-------
+
+The project is licensed under the GPLv2.

--- a/README.rst
+++ b/README.rst
@@ -2,3 +2,6 @@ Mastering Plone
 ===============
 
 Read the trainings at http://training.plone.org
+
+Build the training documentation locally:
+https://training.plone.org/5/mastering_plone/about_mastering.html#building-the-documentation-locally

